core/templates/pages/exploration-editor-page/services/exploration-rights.service.spec.ts

```diff
@@ -279,6 +279,20 @@ describe('Exploration rights service', function() {
       sampleDataResults.rights.viewable_if_private);
   });
 
+  it('should callback to be called in saveModeratorChangeToBackend',
+    function() {
+      $httpBackend.expectPUT('/createhandler/moderatorrights/12345').respond(
+        200, sampleDataResults);
+      const onUnpublishingcallbackFunc =
+        jasmine.createSpy('onUnpublishingcallback');
```

Comment:
```suggestion
      const onUnpublishingcallbackFunc = jasmine.createSpy(
        'onUnpublishingcallback');
```

Break after parenthesis

Reply:
DONE